src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheckTestRule.java

```diff
@@ -116,7 +125,7 @@
             @Node(coordinates = @Loc(value = TEST_2)),
             @Node(coordinates = @Loc(value = TEST_3)) }, edges = {
                     @Edge(coordinates = { @Loc(value = TEST_1), @Loc(value = TEST_2) }, tags = {
-                            "highway=service", "aeroway=taxiway" }),
+                            "highway=service" }),
```

Comment:
...and the change here?

Reply:
Since ways with "aeroway" tags are ingested as lines, I removed this tag from the test data for this edge.